src/OrbitGl/MultivariateTimeSeries.h

```diff
@@ -136,7 +133,7 @@ class MultivariateTimeSeries {
   }
 
   mutable absl::Mutex mutex_;
-  std::map<uint64_t, std::array<double, Dimension>> time_to_series_values_ ABSL_GUARDED_BY(mutex_);
+  absl::btree_map<uint64_t, std::array<double, Dimension>> time_to_series_values_ ABSL_GUARDED_BY(mutex_);
```

Comment:
I think this line is causing the clang-format build to not work, you have to run clang-format.